spyderlib/config/main.py

```diff
@@ -580,6 +580,53 @@ def is_ubuntu():
               # ---- Profiler (in p_profiler) ----
               'profiler/run profiler': "F10"
               }),
+            ('emacs_shortcuts',
+              {
+              # -- In codeeditor
+              'editor/code completion': 'Meta+Tab',
+              'editor/go to definition': "Ctrl+G",
+              'editor/start of line': "Ctrl+A",
+              'editor/end of line': "Ctrl+E",
+              'editor/previous line': "Ctrl+P",
+              'editor/next line': "Ctrl+N",
+              'editor/previous char': "Ctrl+B",
+              'editor/next char': "Ctrl+F",
+              'editor/previous word': "Meta+B",
+              'editor/next word': "Meta+F",
+              'editor/kill to line end': "Ctrl+K",
+              'editor/kill to line start': "Ctrl+U",
+              'editor/yank': 'Meta+Y',
+              'editor/rotate kill ring': 'Shift+Ctrl+Y',
+              'editor/kill previous word': 'Ctrl+Backspace',
+              'editor/kill next word': 'Ctrl+Delete',
+              'editor/start of document': 'Meta+Left',
+              'editor/end of document': 'Meta+Right',
+              # -- In widgets/editor
+              'editor/inspect current object': 'Ctrl+X, Ctrl+I',
+              'editor/go to line': 'Ctrl+X, Ctrl+L',
+              'editor/go to previous file': 'Ctrl+Tab',
+              'editor/go to next file': 'Ctrl+Shift+Tab',
+              '_/file switcher': 'Meta+Shift+P',
+              # -- In spyder.py
+              'editor/find text': "Ctrl+S",
+              'editor/find next': "F3",
+              'editor/find previous': "Ctrl+R",
+              'editor/replace text': "Ctrl+X, Ctrl+H",
+              'editor/undo': 'Meta+Backspace',
+              'editor/redo': 'Ctrl+X, Ctrl+Y',
+              'editor/cut': 'Ctrl+W',
+              'editor/copy': 'Ctrl+C',
+              'editor/paste': 'Ctrl+Y',
+              'editor/delete': 'Delete',
+              'editor/select all': "Ctrl+X+H",
+              # -- In plugins/editor
+              'editor/new file': "Ctrl+X, Ctrl+N",

```

Comment:
Let's leave this one in `Ctrl+N` because Emacs doesn't have a shortcut for new files